phpBB/phpbb/avatar/driver/upload.php

```diff
@@ -89,11 +90,16 @@ public function prepare_form($request, $template, $user, $row, &$error)
 			return false;
 		}
 
-		$template->assign_vars(array(
-			'S_UPLOAD_AVATAR_URL' => ($this->config['allow_avatar_remote_upload']) ? true : false,
-			'AVATAR_UPLOAD_SIZE' => $this->config['avatar_filesize'],
+		$use_board = defined('PHPBB_USE_BOARD_URL_PATH') && PHPBB_USE_BOARD_URL_PATH;
```

Comment:
Slightly more readable name:
```suggestion
		$use_board_url = defined('PHPBB_USE_BOARD_URL_PATH') && PHPBB_USE_BOARD_URL_PATH;
```